### ISSUE TITLE
Suppressing Warnings results in suppressing import module error 

### ISSUE THREAD

**Issue by davidraleigh** [PyCQA/pylint#1533]
### Steps to reproduce
for this code block
```python
import numpy as np
import math
def ndi4_unit16(in_ar, out_ar):
    testing = 0
    if testing == 1:
        a = 2
    out_ar[:] = in_ar[0]
```

run allowing Warnings and Errors:
```bash
pylint --disable=C,R temp.py 
```
Results have both Warnings and Errors:
```bash
************* Module temp
E:  1, 0: Unable to import 'numpy' (import-error)
W:  6, 8: Unused variable 'a' (unused-variable)
W:  1, 0: Unused numpy imported as np (unused-import)
W:  2, 0: Unused import math (unused-import)
```

run it only allowing Errors:
```bash
pylint --disable=C,R,W temp.py 
```

Results have no Warnings but the Error has also been removed:
```bash
----------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: -14.29/10, +24.29)
```

### Current behavior
It is missing the module import error when suppressing warnings

### Expected behavior
Module import error should show up when suppressing warnings

### pylint --version output
```bash
No config file found, using default configuration
pylint 1.7.1, 
astroid 1.5.3
Python 3.6.1 (default, Jun  8 2017, 21:43:55) 
[GCC 4.9.2]
```

**Comment by PCManticore** [comment]
I can reproduce it, thank you!
